src/components/mdTable/mdTablePagination.vue

```diff
@@ -48,7 +48,7 @@
         subTotal: 0,
         currentSize: parseInt(this.mdSize, 10),
         currentPage: parseInt(this.mdPage, 10),
-        totalItems: !isNaN(this.mdTotal) && Number.MAX_SAFE_INTEGER
+        totalItems: isNaN(this.mdTotal)?Number.MAX_SAFE_INTEGER:parseInt(this.mdTotal, 10)
```

Comment:
I see , but i think this change is still necessary , because 
if the user pass a `NAN String` to `mdTotal `, the `totalItems` will return `false` and the `disable` condition in line 15 (e.g 10*5 >= false) will always be `true` .  
if the user pass a `numberlike String` to `mdTotal` , the `totalItems` will return `MAX number` and the `disable` condition will always be `false`